framework/assets/yii.js

```diff
@@ -329,7 +329,7 @@ yii = (function ($) {
             }
 
             if (message !== undefined) {
-                pub.confirm(message, function () {
+                $.proxy(pub.confirm, this)(message, function () {
                     pub.handleAction($this, event);
```

Comment:
if use `$this` in the `yii.confirm` function, we no need to wrap `this` and change to jquery dom object.